numba/targets/base.py

```diff
@@ -487,22 +487,18 @@ def imp(context, builder, typ, val):
         if isinstance(typ, types.Module):
             # Implement getattr for module-level globals.
             # We are treating them as constants.
-            # XXX We shouldn't have to retype this
```

Comment:
Will put that comment back. Is it suggesting that the typing context should hold a mapping from attrributes to types so that we can look things like this up instead of calling `resolve_module_constants` here?